src/ui/page/booking/confirmed.html

```diff
@@ -20,6 +21,10 @@ <h4 class="heading-small">What happens next?</h4>
                     Once confirmed you will be notified and we will contact you with the
                     course logistics.
                 </p>
+            {{else}}
+                <p>Your course booking has been cancelled.</p>
+                <p><a href="/profile">Visit your profile</a></p>
```

Comment:
Why would you go to your profile from this page? Wouldn't you want to go back to home?

Reply:
this is from current csl 😧should we redirect to home?
<img width="855" alt="screen shot 2018-03-05 at 15 30 35" src="https://user-images.githubusercontent.com/22080510/36983415-3580c284-208a-11e8-9c91-88d1950f1710.png">

Reply:
It's possible their profile is more like our home? I think we take them home - just like on booking request submitted there's a "You can now return to the home page" link.